build.gradle

```diff
@@ -9,6 +9,7 @@ plugins {
     id("com.android.library") version "8.7.1" apply false
     id("com.android.application") version "8.7.1" apply false
     id("io.github.gradle-nexus.publish-plugin") version "1.3.0"
+    id 'org.jetbrains.kotlin.android' version '2.2.0' apply false
```

Comment:
Can you use the same Kotlin version we use inside react-native here (2.1.20)?

Reply:
Changed it in a4286e3233e491175fa8d0599880dca97c05fe4a